Jenkinsfile

```diff
@@ -1149,16 +1150,115 @@ pipeline {
                                                   ' cart-' + env.CART_COMMIT + ' ' +
                                                   functional_rpms
                         runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
-                                script: '''test_tag=$(git show -s --format=%B | sed -ne "/^Test-tag-hw:/s/^.*: *//p")
+                                script: '''test_tag=$(git show -s --format=%B | sed -ne "/^Test-tag-hw-large:/s/^.*: *//p")
```

Comment:
Hrm.  I wonder if [coimmitPragma](https://github.com/daos-stack/pipeline-lib/blob/master/vars/commitPragma.groovy) can be used here.

Reply:
I took a quick peak at [commitPragma](https://github.com/daos-stack/pipeline-lib/blob/master/vars/commitPragma.groovy) and [runTest](https://github.com/daos-stack/pipeline-lib/blob/master/vars/runTest.groovy) and it doesn't look like a simple change.  I agree it would be nice to use.

Reply:
Wouldn't something like:

```
                                script: 'test_tag=' + commitPragma(pragma: 'Test-tag-hw-small') + '''
￼                                           if [ -z "$test_tag" ]; then
```

work?